grid2op/tests/test_RewardAlertCostScore.py

```diff
@@ -0,0 +1,373 @@
+        """ When no blackout and no attack occur, and no alert is raised we expect a reward of 0
+            until the end of the episode where we get the max reward 1 as score.
+
+        Raises:
+            Grid2OpException: raise an exception if an attack occur
+        """
+        with grid2op.make(
+            self.env_nm,
+            test=True,
+            difficulty="1",
+            reward_class=_AlertTrustScore
+        ) as env:
+            env.seed(0)
+            env.reset()
+
+            done = False
+            for i in range(env.max_episode_duration()):
+                obs, reward, done, info = env.step(env.action_space())
+                if done:
+                    assert np.round(reward,3) == 1., f"{reward} vs 1."
+                else:
+                    assert reward == 0., f"{reward} vs 0."
+                    
+    def test_assistant_reward_value_blackout_attack_no_alert(self) -> None :
+        """
+        When 1 line is attacked at step 3 and we don't raise any alert
+        and a blackout occur at step 4
+        we expect a score of -10 at step 4 
+        """
+        kwargs_opponent = dict(lines_attacked=[ATTACKED_LINE], 
+                               duration=3, 
+                               steps_attack=[3])
+        with grid2op.make(self.env_nm,
+                  test=True,
+                  difficulty="1", 
+                  opponent_attack_cooldown=0, 
+                  opponent_attack_duration=99999, 
+                  opponent_budget_per_ts=1000, 
+                  opponent_init_budget=10000., 
+                  opponent_action_class=PlayableAction, 
+                  opponent_class=TestOpponent, 
+                  kwargs_opponent=kwargs_opponent, 
+                  reward_class=_AlertTrustScore,
+                  _add_to_name="_tarvbana"
+            ) as env : 
+            new_param = Parameters()
+            new_param.MAX_LINE_STATUS_CHANGED = 10
+
+            env.change_parameters(new_param)
+            env.seed(0)
+            env.reset()
+            step = 0
+            for i in range(env.max_episode_duration()):
+                attackable_line_id = 0
+                act = self.get_dn(env)
+                if step == 3 :
+                    act = self.get_blackout(env)
+                obs, reward, done, info = env.step(act)
+                step += 1
+                
+                if step in _get_steps_attack(kwargs_opponent): 
+                    assert info["opponent_attack_line"] is not None, f"no attack is detected at step {step}"
+                else:
+                    assert info["opponent_attack_line"]  is None, f"an attack is detected at step {step}"
+                
+                if step == 4: 
+                    # When the blackout occurs, reward is -1 because we didn't raise an attack so min
+                    assert reward == env._reward_helper.template_reward.min_score, f"error for step {step}: {reward} vs -1"
```

Comment:
don't rely on data that you want to test. Write clearly what is it supposed to be (-1 if I followed correctly here)

Reply:
test_RewardAlertCostScore.py is disregarded for now